compiler/src/dotty/tools/backend/sjs/JSExportsGen.scala

```diff
@@ -42,6 +43,254 @@ final class JSExportsGen(jsCodeGen: JSCodeGen)(using Context) {
   import jsCodeGen._
```

Comment:
I noticed the scala 2 implementation is named `GenJSExports` and this one is named `JSExportsGen`. Was this change intentional?

Reply:
Yes, it was more consistent with `JSCodeGen`.